source/_components/sensor.thinkingcleaner.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "ThinkingCleaner"
+description: "Instructions how to integrate a ThinkingCleaner within Home Assistant."
+date: 2016-04-10 17:24
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: thinkingcleaner.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
```

Comment:
This should be Local Poll, as HA is initiating state updates from ThinkingCleaner.